channels-rke2.yaml

```diff
@@ -261,6 +313,45 @@ releases:
       rke2-canal:
         repo: rancher-rke2-charts
         version: v3.20.3-build2022011406
+  - version: v1.21.10+rke2r1
+    minChannelServerVersion: v2.6.1-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v1-21-10-rke2r1
+      <<: *serverArgs-v1-21-9-rke2r1
+    agentArgs: &agentArgs-v1-21-10-rke2r1
+      <<: *agentArgs-v1-21-9-rke2r1
+    charts: &charts-v1-21-10-rke2r1
+      <<: *charts-v1-21-9-rke2r1
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.21.4-build2022020801
```

Comment:
I think we need to remove this version. We shouldn't be releasing two v1.21.10 versions since this one hasn't already previously been released.
Note that we resolved this same thing for 1.22.7 in https://github.com/rancher/kontainer-driver-metadata/pull/861.

I will make the change and submit a PR to dev-v2.6 for this.